clients/src/test/java/org/apache/kafka/clients/admin/MockAdminClient.java

```diff
@@ -916,6 +925,34 @@ synchronized private Throwable handleIncrementalResourceAlteration(
                 topicMetadata.configs = newMap;
                 return null;
             }
+            case CLIENT_METRICS: {
+                String subscriptionName = resource.name();
```

Comment:
Again, `resourceName` I think is better.

Reply:
Done.